.github/workflows/jestCI.yml

```diff
@@ -49,6 +49,12 @@ jobs:
 
     steps:
       - uses: actions/checkout@v4
+      - name: Login to GHCR (GitHub Packages)
+        uses: docker/login-action@v3
+        with:
+          registry: ghcr.io
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Why do you need to login here, this workflow isn't doing any pushing..? I don't think you need to log in to pull.